packages/address/src/AddressFormatter.ts

```diff
@@ -29,12 +33,18 @@ export default class AddressFormatter {
     return loadCountry(this.locale, countryCode);
   }
 
-  async getCountries(): Promise<Country[]> {
-    const cachedCountries = ORDERED_COUNTRIES_CACHE.get(this.locale);
+  async getCountries(options?: LoadCountriesOptions): Promise<Country[]> {
+    const includeHiddenZones = options?.includeHiddenZones || false;
```

Comment:
Nitpick: You can do the destructuring in the parameter arguments. This would also allow you to remove the LoadCountriesOptions. as the booleanness of `includeHiddenZones` can be inferred


```suggestion
  async getCountries({includeHiddenZones = false} = {}): Promise<Country[]> {
```